packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -1,9 +1,6 @@
 import React, { forwardRef, ReactElement, Ref } from 'react';
```

Comment:
Moved renderOverlay to it's own component

Reply:
Much better 💪